server/chat-plugins/uno.ts

```diff
@@ -286,46 +264,39 @@ class UnoGame extends Rooms.RoomGame {
 		return name;
 	}
 
-	/**
-	 * @param {string} msg
-	 * @param {boolean} [overrideSuppress]
-	 */
-	sendToRoom(msg, overrideSuppress = false) {
+	sendToRoom(msg: string, overrideSuppress: boolean = false) {
```

Comment:
```suggestion
	sendToRoom(msg: string, overrideSuppress = false) {
```

This is inferrable by the compiler.

Reply:
i know but id prefer to keep it defined in the case that something is added later on that introduces potential complexities